### DISCUSSION TITLE
MDXProvider is not apply Css to its children

### DISCUSSION THREAD

**Discussion by SyedImam1998** [mdx-js/mdx#2244] [Q&A]
Hi Team,
 Can someone please help me here? MDXProvider is not applying the Css to its children 
 
 Go to path: /ola (after opening codesandbox)
 codesandbox Url: [See here](https://codesandbox.io/s/tender-perlman-l3x7m1?file=/pages/Comp.js)

**Comment by wooorm** [comment]
You don’t configure your provider.
a) you don’t need providers: https://mdxjs.com/docs/using-mdx/#mdx-provider
b) you need to configure it if you do use it (see point 2 there, about `options.providerImportSource`)

**Reply by SyedImam1998** [reply]
Can you please guide me seeing this code sandbox URL where should i add this options.providerImportSource.

**Reply by wooorm** [reply]
`next.config.mjs`, where it shows some commented out options (e.g., `jsxImportSource`).

**Reply by SyedImam1998** [reply]
i have added it still didn't work...
![image](https://user-images.githubusercontent.com/52135949/216666527-97e8a321-6a1b-4db6-986e-1837b6183778.png)

**Reply by wooorm** [reply]
That shows `jsxImportSource`, i said `providerImportSource`. And you have to set it to a valid value. See the docs I linked.
